apps/qubit/modules/actor/actions/browseAction.class.php

```diff
@@ -375,8 +375,10 @@ protected function setFilterTags($request)
         // Set label for has digital object filter tag
         if (filter_var($request->hasDigitalObject, FILTER_VALIDATE_BOOLEAN)) {
             $this->setFilterTagLabel('hasDigitalObject', $this->i18n->__('With digital objects'));
-        } else {
+        } elseif (!filter_var($request->hasDigitalObject, FILTER_VALIDATE_BOOLEAN) && '0' == $request->hasDigitalObject) {
```

Comment:
The `!filter_var($request->hasDigitalObject, FILTER_VALIDATE_BOOLEAN) &&` should be removed from this condition. If `filter_var($request->hasDigitalObject, FILTER_VALIDATE_BOOLEAN)` is true, then this condition will not be tested because the first `if (filter_var($request->hasDigitalObject, FILTER_VALIDATE_BOOLEAN)) {` will evaluate true.

Also, the strict comparison operator (===) should be used in `'0' === $request->hasDigitalObject`